NDcPP_v2_2e.adoc

```diff
@@ -2118,32 +2179,118 @@ _The preferred method for verification is the Subject Alternative Name using DNS
+
+_This extension is not required, because older TLS implementations did not negotiate FFC parameters (e.g. Group 14)._|_May select – at least one ffdhe group shall be selected_
+|===
+
+_Subsequent versions of this cPP will require support for the Supported Groups extension and prohibit use of legacy DH groups for key establishment._
+
+*FCS_DTLSC_EXT.1.5* The TSF shall [selection:
+
+* _present the signature_algorithms extension with support for the following algorithms: +
+[selection:_
+
+** _rsa_pkcs1 with sha256(0x0401),_
+** _rsa_pkcs1with sha384(0x0501),_
+** _rsa_pkcs1 with sha512(0x0601),_
+** _ecdsa_secp256r1 with sha256(0x0403),_
+** _ecdsa_secp384r1 with sha384(0x0503),_
+** _ecdsa_secp521r1 with sha512(0x0603),_
+** _rsa_pss_rsae with sha256(0x0804),_
+** _rsa_pss_rsae with sha384(0x0805),_
+** _rsa_pss_rsae with sha512(0x0806),_
+** _rsa_pss_pss with sha256(0x0809),_
+** _rsa_pss_pss with sha384(0x080a),_
+** _rsa_pss_pss with sha512(0x080b)_
+** ] and no other algorithms;
+
+* _present the signature_algorithms_cert extension with the following Signature Schemes:_ +
+_[selection:_
+
+** _rsa_pkcs1 with sha256(0x0401),_
+** _rsa_pkcs1with sha384(0x0501),_
+** _rsa_pkcs1 with sha512(0x0601),_
+** _ecdsa_secp256r1 with sha256(0x0403),_
+** _ecdsa_secp384r1 with sha384(0x0503),_
+** _ecdsa_secp521r1 with sha512(0x0603),_
+** _rsa_pss_rsae with sha256(0x0804),_
+** _rsa_pss_rsae with sha384(0x0805),_
+** _rsa_pss_rsae with sha512(0x0806),_
+** _rsa_pss_pss with sha256(0x0809),_
+** _rsa_pss_pss with sha384(0x080a),_
+** _rsa_pss_pss with sha512(0x080b)_
+** _] and no other SignatureSchemes;_
+
+_]._
+
+*_Application Note {counter:appnote_count}_*
+
+_The selected algorithms and SignatureSchemes must be consistent with FCS_COP.1/SigGen and FCS_COP.1/Hash._
+
+_DTLS 1.2 and DTLS 1.3 define different signature_algorithms extension formats. DTLS 1.2 specifies the HashAlgorithm and SignatureAlgorithm independently while DTLS 1.3 specifies SignatureSchemes. This element intentionally limits the signature and hash algorithms that may be selected_
+
+_The following table provides the ST author with instructions for completing the selections._
+
+[cols="1,1,1"]
+|===
+| |_present the signature_algorithms extension_|_present the signature_algorithms_cert extension_
+|_DTLS 1.3_|_Shall select_|_May select_
+|_DTLS 1.2 without DTLS 1.3_|_May select_|_Shall not select_
+|===
+
+
+*FCS_DTLSC_EXT.1.6* The TSF [selection: _provides, does not provide_] the ability to configure the list of supported ciphersuites as defined in FCS_DTLSC_EXT.1.1.
+
+*_Application Note {counter:appnote_count}_*
+
+_The option 'provides' shall be selected if the TOE provides the ability to configure the list of ciphers as defined in FCS_DTLSC_EXT.1.1 (e.g. enabling/disabling of ciphers, ordering, assigning priorities). Otherwise the option 'does not provide' shall be selected._
+
+*FCS_DTLSC_EXT.1.7* The TSF shall prohibit the use of the following extensions:
```

Comment:
@plughy2 Please review FCS_DTLSC_EXT.1.7 in the NDcPP proposal doc v13d for italics within this SFR.

Reply:
FCS_DTLSC_EXT.1.7:  Made the text italicized where required.